runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperConfig.java

```diff
@@ -191,6 +191,17 @@ default boolean swapTeleportItem()
 
 	@ConfigItem(
 		position = 14,
+		keyName = "swapAbyssTeleport",
+		name = "Teleport to Abyss",
+		description = "Swap Talk-to with Teleport for the Mage of Zamorak."

```

Comment:
NABD - but I’d remove the full stop at the end of the description to keep it concise with other config options